Content.Client/CrewManifest/UI/CrewManifestSection.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Numerics;
+using Content.Shared.CrewManifest;
+using Content.Shared.StatusIcon;
+using Robust.Client.GameObjects;
+using Robust.Client.UserInterface.Controls;
+using Robust.Shared.Prototypes;
+
+namespace Content.Client.CrewManifest.UI;
+
+public sealed class CrewManifestSection : BoxContainer
+{
+    [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+    [Dependency] private readonly IEntitySystemManager _entitySystem = default!;
+    private readonly SpriteSystem _spriteSystem = default!;
+
+    public CrewManifestSection(string sectionTitle, List<CrewManifestEntry> entries)
+    {
+        IoCManager.InjectDependencies(this);
```

Comment:
Don't think you need dependency fields when they're only used in the ctor.

Reply:
Done